rules/write-a-good-pull-request/rule.md

```diff
@@ -77,27 +77,40 @@ Having the **"What"** information allows the reviewers to quickly understand wha
 
 The PR body is a medium for the developer to tell the reviewers what the changes are about. 
 
-::: info
-**Tip:** For straight-forward changes the self-explanatory title might be enough, so description is not really necessary, but be careful when doing this because the reviewer may not have the same familiarity with the project as you.
+::: greybox
+**PR title:** Update Rule “meaningful-pbi-titles/rule”
+
+**PR description:**  
+:::
+::: bad
+Figure: Bad example - Cannot tell what was done here
 :::
 
-Things that need to be kept in mind before writing a Pull Request body:
+::: greybox
+**PR title:** Update Rule “meaningful-pbi-titles/rule”
 
-* What the PR is about and why did you raise it
```

Comment:
@GordonBeeming 

Shouldn't we keep that list?

Reply:
Thanks for catching that, I meant to only remove up to the end of the screenshot